tests/Console/Command/SelfUpdateCommandTest.php

```diff
@@ -391,7 +392,7 @@ private function createToolInfo($isInstalledAsPhar = true)
 
     private function getToolPath()
     {
-        return "{$this->root->url()}/php-cs-fixer";
+        return "{$this->root->url()}/PHP CS Fixer";
```

Comment:
too greedy replacement here?

ps: if this was my project, I would say such changeset needs to be in separate PR. too many logical changes in a single PR is never good. should be 1 PR per 1 feature. also each change should be one commit, so it would be easier to remove commit if it's rejected by a review.

Reply:
and commits should be atomic:
- https://www.freshconsulting.com/atomic-commits/

so change commit like "fix tests" would never exist, you "fix tests" in the commit that broke it. such fixup based flow I have found this helper very useful:
- https://github.com/keis/git-fixup

Reply:
yes indeed, nice catch :+1:

Reply:
about the amount of changes and commits and PR's, I agree, yet I balance against velocity and being practical as well as there is a lot people want from this project and only a limited amount of time to do it

Reply:
@SpacePossum in this specific case, if you would stage that "Fix tests" into main commit (that introduced the breakage), and add it as `git fixup`, and later `git rebase -i --autosquash`, it's very easy to extract commits and move them to separate branch (and PR).

so it's already a different kind of development model, where you fix commits, not add and add commits.

for developing (and checking CI), it's easier if the commits are in a single branch.